### DISCUSSION TITLE
2D Ultrasound Segmentation

### DISCUSSION THREAD

**Discussion by alehic173** [Project-MONAI/MONAILabel#1340] [Q&A] [answered]
Hello,

Bear with me, I am pretty novice in this field. I am trying to segment out bone in some 2D ultrasound datasets using 3D Slicer and MONAILabel. The images I am working with are 0 to 255 grayscale pngs (although I can convert them to mha files). I would like to manually segment bone using MONAILabel, create labels, and train a model. But I am not sure what model to use or if it is even possible. 

I have successfully installed both 3D Slicer and MONAILabel and the server does run without errors. I just am not sure where to go from there. 

Thank you!

**Comment by tangy5** [comment, marked as answer]
You could manually annotate the target just as you show above, once finished labeling, you could click "submit label" button to save the label. Till there are  bunches of labels save, you could click training to train a segmentation model. 

But before that, you need to choose or create a model, using radiology app. But there are no existing 2D ultrasound models, you might need to create a customized app with a defined network, transforms, etc...

**Reply by alehic173** [reply]
Hello, Could you delete this discussion post? Thank you!

**Reply by tangy5** [reply]
You want to delete the entire discussion #1340 ?

**Reply by lassoan** [reply]
This discussion is useful for anyone who wants to get started with ultrasound segmentation, so please don't delete it.

@alehic173 You could ask for deleting any specific sensitive information that you accidentally shared, but I don't see anything like that here.

**Reply by alehic173** [reply]
Could you just delete the images I shared? I edited them out but can still see them in the "edited" version. Thank you!

**Reply by tangy5** [reply]
Thank you. I should deleted the version that with the data, let me know if that is still visible. Feel free to shot more questions.

**Comment by LeidyMoraV** [comment]
Hello, I'm having a similar issue as you, I wonder if you trained the model for the ultrasound bone segmentation using MONAI?

**Reply by lassoan** [reply]
Tamas Ungi has been training ultrasound bone segmentation models for several years, more recently using MONAI. You can reach him - and many others who use ultrasound for image-guided therapy - by posting your question on the 3D Slicer forum (https://discourse.slicer.org).
